crates/matrix-sdk/src/http_client/native.rs

```diff
@@ -102,6 +102,15 @@ impl HttpClient {
                     .record("status", status_code.as_u16())
                     .record("response_size", response_size.to_string_as(true));
 
+                // Record interesting headers.
```

Comment:
Let's add some comment that, if you are adding more headers, ensure that they are not confidential. To lower the chance that we're going to start logging the auth header.